go.mod

```diff
@@ -1,8 +1,9 @@
 module github.com/PDOK/geopackage-optimizer-go
 
-go 1.17
+go 1.22
```

Comment:
go 1.23 is beschikbaar
dus kunnen we niet meteen upgraden daar naartoe

Reply:
zou idd voor 1.23 gaan

Reply:
✅